pkgs/build-support/kernel/make-initrd-ng/src/main.rs

```diff
@@ -85,12 +83,25 @@ fn copy_file<P: AsRef<Path> + AsRef<OsStr>, S: AsRef<Path>>(
             }
         }
         if !found {
-            // glibc makes it tricky to make this an error because
+            // glibc makes this tricky because
             // none of the files have a useful rpath.
-            println!("Warning: Couldn't satisfy dependency {} for {:?}", line, OsStr::new(&source));
+            if !line.starts_with("libc.so.") && !line.starts_with("ld-linux-") {
```

Comment:
I mean it's still not quite right, right? If any glibc lib depends on another other than `libc.so`, then this will break. You can see why I made this a warning to begin with :P